src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -40,6 +40,7 @@ public class Transaction : IEquatable<Transaction>, IInventory, IInteroperable
         private Cosigner[] cosigners;
         private byte[] script;
         private Witness[] witnesses;
+        private UInt256 oracleRequestTx;
```

Comment:
We create a new field! Why not use attribute?

Reply:
https://github.com/neo-project/neo/pull/1539#issuecomment-609380165

Reply:
It's interesting to me to compare this implementation to attribute-based one, for some reason I think that reusing attributes would make things easier and require less changes. And adding this field to each and every transaction is a bit inefficient memory-wise.